CHANGES.md

```diff
@@ -13,6 +13,7 @@ Unreleased changes refer to our current [master branch](https://github.com/openc
 * Fixed `Too many connections to ldap` when openchange runs on samba as member of a domain.
 * Fixed `Mark All as Read` feature regression bug introduced by 7737bdf6
 * Address book working much better than before
+* ResolveNames and ResoveNamesW operations perform a better Ambiguous Name Resolution search, which returns more matches.
```

Comment:
I am for it. Anyway I must note that the specification does not says nothing about the exact ANR search logarithm used so it is a matter that where we define the 'acceptable' and the 'improved' behavior of the search.